src/explore-education-statistics-common/src/modules/table-tool/components/FiltersForm.tsx

```diff
@@ -232,7 +237,31 @@ const FiltersForm = ({
                   {orderedFilters.length > 0 && (
                     <FormFieldset
                       error={getError('filters')}
-                      hint="Select at least one option from all categories"
+                      hint={
+                        <div className="dfe-flex dfe-justify-content--space-between dfe-flex-wrap">
+                          <span className="govuk-!-margin-bottom-2">
+                            Select at least one option from all categories
+                          </span>
+                          {orderedFilters.length > 1 && (
+                            <ButtonText
+                              className="govuk-!-margin-bottom-2"
```

Comment:
Would probably take a leaf out of the Accordion component and add a `aria-expanded` here